fd-policies.md

```diff
@@ -0,0 +1,59 @@
+# File descriptor policies
+
+CRaC requires that the application closes all open files, network connections etc. - on Linux these are represented as file descriptors. However, it might be difficult to alter the application to properly coordinate with the checkpoint, e.g. due to a code in a library you cannot modify. In those cases CRaC offers a limited handling via configuration. Note that this applies only to file descriptors opened through JDK API; anything opened through native code cannot be handled this way.
+
+The configuration is set up by pointing system property `jdk.crac.resource-policies` to a file that consists of several rules separated by three dashes (`---`). Lines starting with hash sign (`#`) are ignored. Each rule consists of several `key: value` pairs. The above is actually a subset of YAML format, so we suggest that you use the `.yaml` or `.yml` extension for convenient use in an editor. See an example of this file:
```

Comment:
Need to mention the format, targets and actions are experimental and are subject to change.